src/transformers/models/t5/modeling_t5.py

```diff
@@ -245,6 +245,9 @@ def forward(self, hidden_states):
         # convert into float16 if necessary
         if self.weight.dtype == torch.float16:
             hidden_states = hidden_states.to(torch.float16)
+        # convert into bfloat16 if necessary
+        elif self.weight.dtype == torch.bfloat16:
+            hidden_states = hidden_states.to(torch.bfloat16)
```

Comment:
```suggestion
        if self.weight.dtype == torch.float16 or self.weight.dtype == torch.bfloat16:
            hidden_states = hidden_states.to(self.weight.dtype)
```
I'd suggest to simplify

Reply:
I think I missed the comment a line 245